person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+  # Getters and setters
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    of_age? || @parent_permission == true
```

Comment:
- [ ] [OPTIONAL] This can be shortened further like so ```of_age? || @parent_permission``` Note that in this scenario, the parent permission can only be true or false. This allows the person to use the services as long as it is true and you do not really have to hard code the "true".

Reply:
Hello @Nemwel-Boniface, thanks for your recommendation, I will apply it.